easybuild/easyconfigs/v/vcflib/vcflib-1.0.2-GCC-10.2.0.eb

```diff
@@ -49,9 +49,13 @@ preconfigopts += " -e 's|multichoose.h|multichoose/multichoose.h|g' -e 's|fileve
 preconfigopts += " -e 's|tabix.hpp|tabixpp/tabix.hpp|g' -e 's|Fasta.h|fastahack/Fasta.h|g'"
 preconfigopts += " -e 's|disorder.h|smithwaterman/disorder.h|g' {} \; && "
 
+postinstallcmds = ["cp -r %(builddir)s/%(name)s-%(version)s/scripts %(installdir)s"]
```

Comment:
Some of the scripts are R scripts. Do we need R as a dependency and checking whether we have the required R packages available?

Reply:
I would say yes... Are you up for looking into this too @jfgrimm?

Reply:
will require moving this and reverse dependents to `foss`, will update

Reply:
That's actually a reason *not* to include `R` as a dependency imho, that kind of change is way too intrusive imho...

If people want to run the R scripts, they should just load `R` too?

Reply:
By adding `R` as a dep we make sure there is no module incompatibility.

Reply:
@branfosj True, but that implies moving it from `GCC` to `foss`, which is a breaking change... Is that worth it in this case?

Reply:
Though there is also https://github.com/easybuilders/easybuild-easyconfigs/issues/15195 as an alternative

Reply:
@jfgrimm I'd say that's a mistake in `snpEff`, we could easily add an easyconfig for snpEff there.

But let's take a step back here, because this is getting messy (cfr. the conflicts that are getting introduced)...

I think we should open two separate PRs:

* One that only also installs the scripts and updates `$PATH` accordingly, without fiddling with dependencies (which is a bug fix);
* One that adds `R` as a dependency for the most recent `vcflib` easyconfig, along with any additional changes that implies (like moving from `GCC` to `foss`), which is a "breaking" change (and hence we should limit the scope of that as much as possible).

Reply:
See https://github.com/easybuilders/easybuild-easyconfigs/pull/12898#pullrequestreview-666258617 for the `snpEff` Java 13 discussion. Also the unanswered GitHub issue on `snpEff` https://github.com/pcingola/SnpEff/issues/345

Reply:
judging by the [debian packaging](https://salsa.debian.org/med-team/snpeff/-/tree/master/debian), it doesn't really need Java 12, they're just depending on `default-jre`

Reply:
yup, Java 11 should be sufficient:
```
$ java -xf snpEff.jar
$ find . -type f -name "*.class" -exec file {} \; | grep -Eo "(version .*)" | sort | uniq
version 45.3
version 46.0 (Java 1.2)
version 48.0 (Java 1.4)
version 49.0 (Java 1.5)
version 50.0 (Java 1.6)
version 51.0 (Java 1.7)
version 52.0 (Java 1.8)
version 53.0
```